profiles/common/modules/custom/nexteuropa_token/modules/nexteuropa_token_ckeditor/nexteuropa_token_ckeditor.info

```diff
@@ -8,3 +8,4 @@ dependencies[] = nexteuropa_token
 dependencies[] = wysiwyg
 dependencies[] = views
 files[] = views/nexteuropa_token_ckeditor_handler_field_selection.inc
+files[] = views/nexteuropa_token_ckeditor_bean_handler_field_selection.inc
```

Comment:
Actually I think, but to be confirmed, that registry autoload (already in platform) only use .php files, but here, that ctools views plugins is ending with .inc. This is why it's not used with an autoloaded I think.